src/services/activation/services/eMailAddress/index.js

```diff
@@ -0,0 +1,163 @@
+const { authenticate } = require('@feathersjs/authentication').hooks;
+const { iff, isProvider, disallow } = require('feathers-hooks-common');
+const { SC_SHORT_TITLE } = require('../../../../../config/globals');
+const logger = require('../../../../logger');
+
+const {
+	validPassword,
+	blockThirdParty,
+	validateEmail,
+	blockDisposableEmail,
+	trimPassword,
+	hasPermission,
+	checkUniqueAccount,
+} = require('../../hooks/utils');
+
+const {
+	STATE,
+	KEYWORDS: { E_MAIL_ADDRESS },
+	lookupByUserId,
```

Comment:
Codacy found an issue: ['lookupByUserId' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5660869)